std/experimental/allocator/building_blocks/bucketizer.d

```diff
@@ -144,18 +145,18 @@ struct Bucketizer(Allocator, size_t min, size_t max, size_t step)
             b = null;
             return true;
         }
-        if (size >= b.length)
+        if (size >= b.length && b.ptr.alignedAt(a) && expand(b, size - b.length))
```

Comment:
This won't work, because we don't want short-circuit the flow if we aren't able to `expand`. Instead we want to fall back to other options. I will split the `if` on multiple lines.

Reply:
I just noticed that those lines have 81 chars. Should I still split them?

Reply:
We have a soft limit of 80 and a hard limit of 120. So, in general, lines should be trying to be within 80 characters, but plenty of lines in Phobos exceed 80 characters, and there's certainly no reason to break up a line just because it hit 81. You only _need_ to break up a line if it hits 121.